### DISCUSSION TITLE
Gear down selector not working (Thrustmaster Airbus TCA Quadrant)

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#6185] [General]
Hello folks,
my Thrustmaster Airbus TCA Quadrant worked fine for a long time, but since short time ago, I'm NOT able to select gear down.
GEAR UP is binded to function 21. But gear down has no bindable function button in the MS FS device software settings.
(I'm using A32NX Development version with latest updates for everything)
Can anybody help me with bringing back the GEAR DOWN function with the TCA Quadrant gear selector?
Thank's to all from Mike

**Comment by tracernz** [comment]
I think the only way is to duplicate the default profile and adjust the throttle curves only; don't touch the gear and engine selector binds as the defaults seem to use some magic that's not exposed in the UI to recreate if you edit them.
![image](https://user-images.githubusercontent.com/9995998/141939840-824001e6-563f-4be2-912e-1901a731f04a.png)

**Reply by Mickepilot** [reply]
Thank you Tracernz! The "default" profile gives back the gear selector function. I also had to recalibrate the throttles and now everything works as intended, great.
